src/components/Nav/Menu/useNavMenu.ts

```diff
@@ -11,6 +14,25 @@ export const useNavMenu = (sections: NavSections) => {
   const menuColors = useNavMenuColors()
   const [activeSection, setActiveSection] = useState<NavSectionKey | null>(null)
 
+  // Focus corresponding nav section when number keys pressed
+  useEventListener("keydown", (event) => {
+    if (!document || !event.key.match(/[1-9]/)) return
+    if (event.target instanceof HTMLInputElement) return
+    if (event.target instanceof HTMLTextAreaElement) return
+    if (event.target instanceof HTMLSelectElement) return
+
+    const sectionIdx = parseInt(event.key) - 1
+    if (sectionIdx >= SECTION_LABELS.length) return
+
+    const button = document.querySelector(
```

Comment:
One observation: this is not a very react way to do this. We already know all the buttons that are in the menu and we could easily have the reference of each one. This would make all the checking you have to do with this approach a lot easier.

Not a blocker as that works fine too. Just to see what you think, maybe I'm missing something important.

Reply:
Yes, I agree with the overall sentiment... I can look at this again, but I did try this is the initial approach... I forget exactly why at the moment, but I hit some issues passing the refs around and getting the correct ones to trigger correctly... I recall it ended up having a lot more code than this, since this approach just takes advantage of the `nth-of-type` selector. Happy to check this out again, but wanted to just drop a reply.